### DISCUSSION TITLE
[Q&A] 密码只能设置6位吗？

### DISCUSSION THREAD

**Discussion by ravenk1n9** [lobehub/lobehub#575] [General Question] [answered]
文档中写道：
ACCESS_CODE
类型：可选
描述：添加访问 LobeChat 服务的密码，密码应为 6 位数字或字母
默认值：-
示例：awCT74 或 e3@09!

那么密码支持6位以上的密码吗？担心6位密码被破解的概率较高。

**Comment by arvinxx** [comment, marked as answer]
支持的，文档已更新：https://github.com/lobehub/lobe-chat/pull/576

**Comment by chenlikun2010** [comment]
可以设置多个code吗？通过什么格式来设置哦

**Reply by arvinxx** [reply]
#638

**Comment by KZM9** [comment]
怎么设置密码！谢谢
